### DISCUSSION TITLE
having trouble getting the value from decoded global

### DISCUSSION THREAD

**Discussion by lonelybud** [YimMenu/YimMenu#3502] [General]
can someone give me the code to get this integer - `Global_2359296[0].f_5152.f_224[0]`. In the meantime I will also get to learn something.

**Comment by ShinyWasabi** [comment]
I guess you're trying to read the global in this function?
```c
void func_18258(int iParam0)//Position - 0x5A352F
{
	func_130(&(Global_2359296[func_908() /*5570*/].f_5152.f_15[iParam0 /*13*/]));
	Global_2359296[func_908() /*5570*/].f_5152.f_224[iParam0] = 0;
}
```
You would read it like this in YimMenu:
`*script_global(2359296).at(0, 5570).at(5152).at(224).at(0).as<int*>();`
Change last offset to whatever iParam0 is.

**Reply by lonelybud** [reply]
yes this is the actual one `Global_2359296[func_177() /*5570*/].f_5152.f_224[iVar0]);` in `am_contact_requests.c`

**Reply by lonelybud** [reply]
Thank you so much ❤️!  I will try the code.
